draft-irtf-cfrg-vdaf.md

```diff
@@ -149,6 +149,16 @@ informative:
     seriesinfo: S&P 2020
     target: https://eprint.iacr.org/2019/074
 
+  MPRV09:
+    title: "Computational Differential Privacy"
+    author:
+      - ins: I. Mironov
+      - ins: O. Pandey
+      - ins: O. Reingold
+      - ins: S. Vadhan
+    seriesinfo: CRYPTO 2009
+    target: https://link.springer.com/chapter/10.1007/978-3-642-03356-8_8
+
```

Comment:
We can also delete the commented-out "Vad16" reference below. That was just used [here](https://github.com/cjpatton/ppm/blob/1d38aacd713ec7f23a3de3a41c5166e29ef43be2/draft-patton-multi-ppm.md?plain=1#L377) early in the history of the document, and the other references we have are sufficient.